.changeset/strange-fans-impress.md

```diff
@@ -0,0 +1,14 @@
+---
+"@razorpay/blade": minor
+---
+
+feat(Collapsible): mark width prop as external
+
+> [!WARNING]
+>
+> If you were using internal `_width` prop, you will have to rename it to `width`
+
+```diff
+- <CollapsibleBody _width={} />
```

Comment:
I see 17 instances of `_width` prop. this will be a breaking change for all these customers right?
<img width="557" alt="Screenshot 2023-12-12 at 12 39 24 PM" src="https://github.com/razorpay/blade/assets/11384858/589f7145-6179-4c17-a0fa-3cdd3908aef0">

Reply:
@kamleshchandnani can you share link of this chart? I searched on github and it shows 5 files only https://github.com/search?q=org%3Arazorpay+CollapsibleBody+_width+NOT+repo%3Arazorpay%2Fblade&type=code

Also its an internal prop so having a minor release with warning note should be fine right?

Reply:
here: https://app.amplitude.com/analytics/razorpay-mobile/chart/zity1yl/edit/ifsok2i1